### PR TITLE
Integration refactor

### PR DESCRIPTION
**Pull request by konsti219**

### CODE REVIEW

File: unreal_cpp_bootstrapper/src/lib.rs [hunk 1]
```diff
@@ -13,10 +13,17 @@ pub const LOADER_DLL: &[u8] = include_bytes!(env!("CPP_LOADER_DLL_PATH"));
...
     fn get_config_location(&self) -> Result<PathBuf, E>;
-    fn get_extract_path(&self) -> Option<PathBuf>;
+    /// Path to extract Mod DLLs to
+    fn get_extract_path(&self) -> Result<PathBuf, E>;
+    /// Function run before starting the game while Mods are being installed. Used for for writing DLLs.
     fn prepare_load(&self) -> Result<(), E>;
+    /// Function run while the game starts.
+    /// Used to inject DLLs if the install method does not allow for the game to load DLLs by iteself.
     fn load(&self) -> Result<(), E>;
+    /// Remove all files that the loader places somewhere to restore vanilla bahaviour.
```
Comment (localcc) [inline unreal_cpp_bootstrapper/src/lib.rs:25]: maybe change wording from somewhere to anywhere?

Comment (localcc) [reply unreal_cpp_bootstrapper/src/lib.rs:25]: Or maybe change it to "Remove all the files that the loader has placed to restore vanilla behavior"?

Comment (konsti219) [reply unreal_cpp_bootstrapper/src/lib.rs:25]: "Undo all modifications to restore vanilla behaviour"?

Comment (localcc) [reply unreal_cpp_bootstrapper/src/lib.rs:25]: that works

**Pull request merged**